content/en/docs/Overview/_index.md

```diff
@@ -6,11 +6,11 @@ description: >
   All about Parquet.
 ---
 
-Apache Parquet is a columnar storage format available to any project in the Hadoop ecosystem, regardless of the choice of data processing framework, data model or programming language.
+Apache Parquet is an open source, column-oriented data file format designed for efficient data storage and retrieval.
+It provides high performance compression and encoding schemes to handle complex data in bulk and is supported in many programming language and analytics tools.
```

Comment:
Did we mean for this to say "high performance compression" or is it "high performance, compression"? I think it may be the latter

Reply:
I didn't mean for the comma or lack there of to carry any additional semantic meaning. I am happy to put a comma there if you like

Reply:
No really strong feelings, was just wondering if there was a subtextual focus intended